packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -72,11 +73,39 @@ import MAX_SIGNED_31_BIT_INT from './maxSigned31BitInt';
 let didWarnAboutBadClass;
 let didWarnAboutGetDerivedStateOnFunctionalComponent;
 let didWarnAboutStatelessRefs;
+let emptyContextThatWarnsOnAccess;
 
 if (__DEV__) {
   didWarnAboutBadClass = {};
   didWarnAboutGetDerivedStateOnFunctionalComponent = {};
   didWarnAboutStatelessRefs = {};
+
+  emptyContextThatWarnsOnAccess =
```

Comment:
I'm no expert here, but since this is already a `let`, wouldn't using an `if else` block  instead of a ternary make this easier to digest?